.changesets/maint_garypen_router_258_update_router_bridge.md

```diff
@@ -0,0 +1,7 @@
+### Update the version of the router-bridge ([PR #5064](https://github.com/apollographql/router/pull/5064))
+
+Update our router-bridge version to pull in fixes.
+
+The router is now using: router-bridge = "=0.5.21+v2.7.5"
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/5064
```

Comment:
@garypen I don't actually quite understand why this changelog is still here — we released this in 1.46.0.  I had changed your message in that release to map to [what I thought the end-user would understand an appreciate more](https://github.com/apollographql/router/pull/5060/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR110-R137), but strangely I still see it here.

Reply:
Hmm, none of the `.changesets` were removed in the last release!

Reply:
Ok, there's only two hold-overs and  [`52bbe1a` (#5180)](https://github.com/apollographql/router/pull/5180/commits/52bbe1ad54a79904bdafc812dfe130290498eb83) fixes that.  Not sure if you have any perspective on what happened here, but I'm interested!

Reply:
I resolve this conversation to unblock the 1.47.0 release.
Not sure if there is any follow-up after 1.47.0 is needed 🤷